src/measurement_buffer.cpp

```diff
@@ -115,23 +116,21 @@ void MeasurementBuffer::BufferPCLCloud(const \
   try
   {
     // transform into global frame
-    geometry_msgs::Quaternion orientation;
-    tf::Stamped<tf::Pose> local_pose, global_pose;
-    local_pose.setOrigin(tf::Vector3(0, 0, 0));
-    local_pose.setRotation(tf::Quaternion(0, 0, 0, 1));
-    local_pose.stamp_ = pcl_conversions::fromPCL(cloud.header).stamp;
-    local_pose.frame_id_ = origin_frame;
-
-    _tf.waitForTransform(_global_frame, local_pose.frame_id_, \
-                         local_pose.stamp_, ros::Duration(0.5));
-    _tf.transformPose(_global_frame, local_pose, global_pose);
-
-    _observation_list.front()._origin.x = global_pose.getOrigin().getX();
-    _observation_list.front()._origin.y = global_pose.getOrigin().getY();
-    _observation_list.front()._origin.z = global_pose.getOrigin().getZ();
-
-    tf::quaternionTFToMsg(global_pose.getRotation(), orientation);
-    _observation_list.front()._orientation = orientation;
+    geometry_msgs::PoseStamped  local_pose, global_pose;
+    //local_pose.setOrigin(tf::Vector3(0, 0, 0));
```

Comment:
Can you readd these lines? I'm a strong believer that a strong offense for stupid programming (for me) is a defense for myself.

Reply:
Not sure what you meant but I removed the commented lines